test/integration/sagemaker/test_mnist.py

```diff
@@ -146,6 +146,26 @@ def test_tuning(sagemaker_session, ecr_image, instance_type, framework_version):
         tuner.wait()
 
 
```

Comment:
Needs this test fixture: 
```
@pytest.mark.skip_py2_containers
```

Reply:
is this required in tf2 branch as well?

Reply:
Yes. There are py2 containers for tf2, so this will be required for tf2 as well.